src/compiler/builderState.ts

```diff
@@ -76,20 +78,78 @@ namespace ts {
             signature: string | undefined;
             affectsGlobalScope: boolean | undefined;
         }
-        /**
-         * Referenced files with values for the keys as referenced file's path to be true
-         */
-        export type ReferencedSet = ReadonlySet<Path>;
+
+        export interface TwoWayMap<K, V> extends ESMap<K, ReadonlySet<V>> {

```

Comment:
Different name would be better:
BiDirectionalFileToReferenceSet or something like that ? atleast alias would be nice to use everywhere.

Reply:
Now [Readonly]ManyToManyPathMap.